src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
-public class MyHashMap<K, V> implements MyMap<K, V> {
+public class MyHashMap<K,V> implements MyMap<K,V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
```

Comment:
I hope you know what does `1 << 4` mean?

Reply:
Yes I am. It's 4 bitwise shift.
The same as 1 * 2 * 2 * 2 * 2 = 1 * 16 = 16

But I can remove it.